### ISSUE TITLE
Output Files are not being updated when switching between components

### ISSUE THREAD

**Issue by fatkaratekid** [elyra-ai/elyra#1687]
**Describe the issue**
A clear and concise description of what the issue is.

When I click on components with defined 'Output Files' the value is not being updated.

**To Reproduce**
Clicked on the first component
![1](https://user-images.githubusercontent.com/2835130/119144492-012c6e00-ba49-11eb-9ee5-25bede0c5891.png)
It shows 'tobi.csv' as it should.

Clicked on the second component
![2](https://user-images.githubusercontent.com/2835130/119144490-012c6e00-ba49-11eb-9bef-42d636bb4a75.png)
It shows 'tobi.csv', it should show 'saved_model/my_model'

Clicked on the third component
![3](https://user-images.githubusercontent.com/2835130/119144489-0093d780-ba49-11eb-9476-a7d03e9475ad.png)
Shows nothing as it should

Clicked on second component
![4](https://user-images.githubusercontent.com/2835130/119144488-0093d780-ba49-11eb-9ba0-646b0097411a.png)
It shows 'saved_model/my_model', as it should.

Clicked on the first component
![5](https://user-images.githubusercontent.com/2835130/119144485-fffb4100-ba48-11eb-88a3-55ef56f17aa9.png)
It shows ' saved_model/my_model', it should show 'tobi.csv'

**Expected behavior**
It should show values that have been given.

**Deployment information**
 - Elyra version: elyra/elyra@sha256:6c4c2178bde39eb2c22ae5ef7b043cf514e67c637e07918ea054b54318adbef8
 - Operating system: linux
 - Deployment type: Kubernetes

**Comment by ptitzler** [comment]
Confirmed this is a bug in the 2.2.x releases. 

simple pipeline [count.tar.gz](https://github.com/elyra-ai/elyra/files/6523777/count.tar.gz)


- switch between nodes and inspect output fields, which are set for three of the four nodes

```
$ grep ".csv" count.pipeline 
              "saved/zwei.csv"
              "vier.csv"
              "eins.csv"
```

**Comment by ptitzler** [comment]
This is already fixed in and available in the **2.3 pre-release** (https://hub.docker.com/layers/elyra/elyra/2.3.0b1/images/sha256-090c975c9afa31f399dbbe18d92c97c8fe283db0556437938a5f64131d181a0b?context=explore). Note that this is not a stable release.
